internal/operators/ocs/validations.go

```diff
@@ -156,7 +156,7 @@ func (o *operator) canOCSBeDeployed(hosts []*models.Host, nodeInfo *resourceInfo
 	}
 
 	if nodeInfo.numDisks%3 != 0 {
-		status = fmt.Sprint("Total disks on the cluster must be a multiple of 3 of size >= ", minDiskSize, " GB")
+		status = fmt.Sprint("Total disks on the cluster must be a multiple of 3 of size >= ", MinDiskSize, " GB")
```

Comment:
Please reword to " The number of disks in the cluster for OCS must be a multiple of 3 with a minimum size of "..